sky/packages/updater/pubspec.yaml

```diff
@@ -1,9 +1,16 @@
-name: sky_updater
+name: updater
 version: 0.0.1
 author: Chromium Authors <sky-dev@googlegroups.com>
 description: The autoupdater for flutter
 homepage: http://flutter.io
 dependencies:
-  mojo: ^0.0.21
+  mojo: '>=0.1.0 <0.2.0'
+  sky: any
+  sky_services: any
+dependency_overrides:
+  sky:
+    path: ../sky
+  sky_services:
+    path: ../sky_services
```

Comment:
You can't use a local dependency override for sky_services because the source tree doesn't contain the generated mojom.dart files.  Instead, you need to pull down the published versions from pub.dartlang.org.  Even though this code and the source code for sky_services is in the same git repository, they can't be changed atomically.